CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -398,6 +398,24 @@ function PlayerHospital:onEndMonth()
   Hospital.onEndMonth(self)
 end
 
+--! Tell the player, through the advisor, about the impact of casebook prices
+--!param judgment (string - under, over, fair) The judgment of the price,
+-- from Hospital:computePriceLevelImpact
+--!param name (string) The name of the casebook entry of the diagnosis or disease
+function PlayerHospital:advisePriceLevelImpact(judgment, name)
+  local message
+  if judgment == "under" then
+    message = _A.warnings.low_prices:format(name)
+  elseif judgment == "over" then
+    message = _A.warnings.high_prices:format(name)
+  elseif judgment == "fair" then
```

Comment:
```suggestion
  else
    assert(judgment == "fair")
```
This ensures the code dies on typos in the `judgment` value.

Reply:
Stylistically I prefer
```lua
if foo == "a" then
   ---
elseif foo == "b" then
  ---
elseif foo == "c" then
  ---
else
  error("foo must be a, b, or c")
end
```
but I suppose it doesn't matter.